cmake/CMakeLists.txt

```diff
@@ -526,6 +527,27 @@ if(PKG_USER-SCAFACOS)
   include_directories(${SCAFACOS_INCLUDE_DIRS})
 endif()
 
+if(PKG_USER-PLUMED)
+  option(DOWNLOAD_PLUMED "Download Plumed (instead of using the system's one)" OFF)
+  if(DOWNLOAD_PLUMED)
+    include(ExternalProject)
+    ExternalProject_Add(plumed_build
+      URL https://github.com/plumed/plumed2/releases/download/v2.4.3/plumed-2.4.3.tgz 
+      URL_MD5 bed9c88c8c635b269c1cd4a0598a2934
+      BUILD_IN_SOURCE 1
```

Comment:
Out-of-source is not support!

Reply:
@junghans please re-check the updated package, which imports Plumed.h via `-I/path/to/plumed2/`

Reply:
Include dir was added.